### ISSUE TITLE
How to connect to IMAP/SMTP after installation?

### ISSUE THREAD

**Issue by brylie** [modoboa/modoboa#1142]
# Impacted versions

* Modoboa:  	1.7.4
* installer used: Yes

# Steps to reproduce
1. complete installer
2. log in as admin user
3. create domain
4. create identity
5. try to configure mail client

# Current behavior
I cannot connect my mail client to the Modoboa server. I am also getting errors when running checks from MXtoolbox.

Thunderbird cannot seem to connect to the server via IMAP or SMTP, while MXtoolbox says it fails to connect to SMTP.

# Expected behavior
There would be clear documentation about how to connect to the mail server with an IMAP/SMTP client.

# Video/Screenshot link (optional)
![screenshot_20170617_165528](https://user-images.githubusercontent.com/17307/27253467-c7265f3a-537d-11e7-9fc9-8aeba28381eb.png)

**Comment by tonioo** [comment]
Have you enabled automx during the installation? Try those settings:
* SMTP: port 587, STARTTLS, normal auth
* IMAP: port 143, STARTTLS, normal auth

**Comment by brylie** [comment]
No, I didn't know to enable automx during the installation.

Thanks for the quick response. Would you consider adding these settings to some end-user facing screen or documentation, so there isn't any guesswork?

**Comment by tonioo** [comment]
Yes, I can add something but I don't know where... FYI, automx is an auto configuration service.

**Comment by brylie** [comment]
Alright, cool. It seems that some of this could be ironed out by an installation 'wizard.

> I can add something but I don't know where

Full disclosure, I set up *mail-in-a-box* today, because I needed an SMTP server ASAP (we are deploying our service for usage tomorrow). *mail-in-a-box* has a full page of usage instructions, which are accessible from the user interface:

![screen shot 2017-06-18 at 16 29 34-fullpage](https://user-images.githubusercontent.com/17307/27261047-d9155b44-5443-11e7-9f09-3679b9bc53df.png)

**Comment by brylie** [comment]
Perhaps usage instructions could be included on the [domain detail](https://github.com/modoboa/modoboa/blob/master/modoboa/admin/templates/admin/domain_detail.html) page in a 'usage' panel?

**Comment by tonioo** [comment]
yes, that's one possible solution :)

**Comment by Nodeswitch** [comment]
We're having the same issue as above. We can connect to the web client for mail, but when I try to connect via Thunderbird, getting nothing.

Ports 25, 587, 143 and 993 are all set to listen and it looks like iptables is set to allow everything by default.

```
tcp        0      0 0.0.0.0:25              0.0.0.0:*               LISTEN      1616/master     
tcp        0      0 0.0.0.0:587             0.0.0.0:*               LISTEN      1616/master     
tcp        0      0 0.0.0.0:143             0.0.0.0:*               LISTEN      1309/dovecot    
tcp        0      0 0.0.0.0:993             0.0.0.0:*               LISTEN      1309/dovecot    
```
Automx was enabled at install. We've tried both automatic settings and using the open ports with different settings for authentication, but getting nowhere.

Running 1.8.1, but upgraded from a version installed back in April. Using a client has never worked, as far as we're aware.

**Comment by edenprojectde** [comment]
Is this still not resolved?

**Comment by rebornweb** [comment]
Modoboa does work on Thunderbird, just make sure you have open Inbound Port 993 open on your firewall/security for (IMAP/SMTP), that your username is: support@blah.com (email) and you have your Outbound ports open too!
